src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java

```diff
@@ -11,12 +11,12 @@
 
 public class InputAvoidDoubleBraceInitialization {
     void m() {
-        ArrayList<?> list = new ArrayList<Object>() { // violation
+        ArrayList<?> list = new ArrayList<Object>() { // violation 'Avoid double brace initialization.'
```

Comment:
Please do not use any messages when Check has single message https://checkstyle.org/config_coding.html#AvoidDoubleBraceInitialization_Violation_Messages

Reply:
done